Server/package.json

```diff
@@ -10,9 +10,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "bcrypt": "^3.0.6",
     "body-parser": "^1.19.0",
     "express": "^4.17.1",
-    "mongoose": "^5.6.0"
+    "jshint-esnext": "^2.7.0-3",
```

Comment:
is this required?

Reply:
Removed , earlier i have used to solve some issue . But now it is not required.